packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1303,6 +1305,12 @@ HTTP/1.1 200 OK
 {
     "name": "UserActivation",
     "_links": {
+    "_embedded": {
+      "settings": {
+        "recipients": "ALL_USERS",
+        "_links": { ... }
+      },
+      "customizationCount": 3
```

Comment:
The `_embedded` block should be a sibling to the `_links` blocks (i.e., at the same level):
```
{
  "name": ...
  "_links": { ... },
  "_embedded": {
    "settings": ...
    "customizationCount": ...
  }
}
```